src/components/Articles/index.tsx

```diff
@@ -0,0 +1,115 @@
+'use client'
+import './styles.scss';
+import image from "./imagen/Image.jpg";
+import Image from "next/image";
+import React, { useState, useEffect } from 'react';
+import { fetchData } from '../../app/apiconection/api';
+import { FaRegFilePdf } from "react-icons/fa6";
+import { GoPaperclip } from "react-icons/go";
+import { FaFacebook, FaTelegram, FaWhatsapp } from 'react-icons/fa6';
+import { TbPointFilled } from "react-icons/tb";
+
+const SliderComponent: React.FC = () => {
+  const [articles, setArticles] = useState<Article[]>([]);
+  const [currentSlide, setCurrentSlide] = useState(0);
+
+  useEffect(() => {
+    const fetchDataAndSetState = async () => {
+      const data = await fetchData();
+      setArticles(data);
+    };
+
+    fetchDataAndSetState();
+  }, []);
+  const [menuVisible, setMenuVisible] = useState(false);
+  const toggleMenu = () => {
+    setMenuVisible((prevMenuVisible) => !prevMenuVisible);
+  };
+  const nextSlide = () => {
+    setCurrentSlide((prevSlide) => (prevSlide + 1) % articles.length);
+  };
+
+  const prevSlide = () => {
+    setCurrentSlide((prevSlide) => (prevSlide - 1 + articles.length) % articles.length);
+  };
+
+  const getCurrentDate = () => {
+    const options = { year: 'numeric', month: 'long', day: 'numeric' };
+    const currentDate = new Date().toLocaleDateString('es-ES', options);
+    return currentDate;
+  };
+
+  return (
+    <div className="flex min-h-screen items-top justify-between p-24">
+      <div className="w-3/4 mr-16 relative">
+        {articles.length > 0 && (
+          <>
+
+            <div className="slider-content">
```

Comment:
you need to separate this code into another component, you will create another component to manage only the articles, and you can also use this as a Wrapper.